Modules/ThirdParty/HDF5/src/itkhdf5/src/H5win32defs.h

```diff
@@ -68,6 +67,15 @@ typedef __int64             h5_stat_size_t;
 
 #ifdef H5_HAVE_VISUAL_STUDIO
 
+/* _O_BINARY must be set in Windows to avoid CR-LF <-> LF EOL
```

Comment:
Does the upstream version of this file fix this issue?

https://github.com/live-clones/hdf5/blob/master/src/H5win32defs.h#L46-L49

Reply:
Yes I believe so.

Reply:
I don't actually believe that this fixes it sufficiently without hte rest of the HDF5 changes as failures happen with `H5Defl.c`:
```
C:\run\ITKR.Rcheck\00_pkg_src\ITKR\src\itks\Modules\ThirdParty\HDF5\src\itkhdf5\src\H5Defl.c: In function 'H5D__efl_read':
C:\run\ITKR.Rcheck\00_pkg_src\ITKR\src\itks\Modules\ThirdParty\HDF5\src\itkhdf5\src\H5Defl.c:291:44: error: macro "HDopen" requires 3 arguments, but only 2 given
         if((fd = HDopen(full_name, O_RDONLY)) < 0)
                                            ^
C:\run\ITKR.Rcheck\00_pkg_src\ITKR\src\itks\Modules\ThirdParty\HDF5\src\itkhdf5\src\H5Defl.c:291:18: error: 'HDopen' undeclared (first use in this function)
         if((fd = HDopen(full_name, O_RDONLY)) < 0)
                  ^
C:\run\ITKR.Rcheck\00_pkg_src\ITKR\src\itks\Modules\ThirdParty\HDF5\src\itkhdf5\src\H5Defl.c:291:18: note: each undeclared identifier is reported only once for each function it appears in
```

https://ci.appveyor.com/project/muschellij2/itkr/builds/23054691

Reply:
This still is an issue, but I think we can bring up in new PR